ItemIdName.h

```diff
@@ -0,0 +1,187 @@
+#pragma once
+enum ItemIdName
```

Comment:
specify enum eplicit underlying type with:
`enum ItemIdName : u16` (I don't know if u16 is appropriate, just assuming